### PR TITLE
[IMP] payment_providers,stripe: rename payment icon to payment method

### PR DESCRIPTION
**Pull request by anko-odoo**
task-2882564

Changing every occurrence of Payment Icon to Payment Method.

### CODE REVIEW

File: content/applications/finance/payment_providers.rst [hunk 1]
```diff
@@ -307,7 +307,7 @@ Configuration tab
 ~~~~~~~~~~~~~~~~~
 
 You can change the payment provider's front-end appearance by modifying its name under the
-**Displayed as** field and which credit card icons to display under the **Supported Payment Icons**
+**Displayed as** field and which credit card icons to display under the **Supported Payment Methods**
```
Comment (toaa-odoo) [inline content/applications/finance/payment_providers.rst:310]:
```suggestion
**Displayed as** field, and the supported credit card icons displayed under the **Supported Payment Methods**
```

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Just a small change, because the sentence was a bit confusing :)

File: content/applications/finance/payment_providers.rst (hunk 1 above)
Comment (anko-odoo) [reply content/applications/finance/payment_providers.rst]: Done :)

**Review by chevalierv — APPROVED** [review]
LGTM :+1:

**Review by toaa-odoo — APPROVED** [review]
LGTM 2

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**